cfme/tests/cloud/test_snapshot.py

```diff
@@ -165,3 +132,30 @@ def test_snapshot_cloud_tenant():
         1685300
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
+@test_requirements.snapshot
+@pytest.mark.tier(2)
+def test_snapshot_description_mandatory():
+    """
+    Test that when creating snapshots on Openstack provider the Description field is mandatory.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Cloud
+        caseimportance: high
+        initialEstimate: 1/4h
+        setup:
+            1. Have OSP provider added and test VM created
+        testSteps:
+            1. Try to create a snapshot with no description
+            2. Try to create a snapshot with description
+        expectedResults:
+            1. Snapshot not created; 'Required' displayed
+            2. Snapshot created successfully
+    Bugzilla:
+        1647916
```

Comment:
@apagac since you are putting in the BZ metadata, can you also add it to the top of the file?
```@pytest.mark.meta(coverage=[<bug_id>])```

Reply:
+ the same for other tests